hyperspy_gui_ipywidgets/axes.py

```diff
@@ -114,7 +114,7 @@ def _get_axis_widgets(obj):
         link((obj, "value"), (value, "value"))
         link((obj, "high_value"), (value, "max"))
         link((obj, "low_value"), (value, "min"))
-        link((obj, "scale"), (value, "step"))
+        #link((obj, "scale"), (value, "step"))
```

Comment:
Why is it commented?

Reply:
The tests for non_uniform_axes failed at this line as this axis types do not contain a `scale` parameter.

Anyway scale is defined a few lines below in case it is part of the parameters. And would then overwrite this line. So it should probably be removed completely?